backend/api.js

```diff
@@ -19,15 +11,37 @@ const foundParams = (params, wantedParam) => {
     return false;
 };
 
-export function v1(requestEvent, sequelizeManager, callback) {
+const obtainDatabaseForTask = (
+    requestEvent, sequelizeManager, sessionSelected, callback
+) => {
+    let database = null;
+    if (foundParams(requestEvent.params, 'database')) {
+        // check if there is a current database in use
+        database = requestEvent.params.database;
+    } else if (sequelizeManager.sessions[sessionSelected]) {
+        database = sequelizeManager.sessions[sessionSelected].config.database;
```

Comment:
You're right, but if I make it strict now it won't work with plotly 2.0 at all. I planned to wait for this to get implemented in plotly 2.0 first and then make it strict. It's a minor change so would not be a big problem.